[2023-10-17T11:19:18.319+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-31T00:00:00+00:00 [queued]>
[2023-10-17T11:19:18.328+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-31T00:00:00+00:00 [queued]>
[2023-10-17T11:19:18.328+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:18.342+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-31 00:00:00+00:00
[2023-10-17T11:19:18.349+0000] {standard_task_runner.py:57} INFO - Started process 612 to run task
[2023-10-17T11:19:18.353+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-31T00:00:00+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpjnqhwk8h']
[2023-10-17T11:19:18.354+0000] {standard_task_runner.py:85} INFO - Job 94: Subtask waiting_for_data
[2023-10-17T11:19:18.390+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-31T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:18.456+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-31T00:00:00+00:00'
[2023-10-17T11:19:18.462+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:18.463+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:18.463+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:18.463+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:18.470+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210331T000000, start_date=20231017T111918, end_date=20231017T111918
[2023-10-17T11:19:18.525+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:18.543+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:42:48.034+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-31T00:00:00+00:00 [queued]>
[2023-10-18T10:42:48.044+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-31T00:00:00+00:00 [queued]>
[2023-10-18T10:42:48.044+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:42:48.092+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-31 00:00:00+00:00
[2023-10-18T10:42:48.100+0000] {standard_task_runner.py:57} INFO - Started process 1576 to run task
[2023-10-18T10:42:48.103+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-31T00:00:00+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpultvwm7u']
[2023-10-18T10:42:48.104+0000] {standard_task_runner.py:85} INFO - Job 93: Subtask waiting_for_data
[2023-10-18T10:42:48.158+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-31T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:42:48.238+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-31T00:00:00+00:00'
[2023-10-18T10:42:48.255+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:42:48.259+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210331T000000, start_date=20231018T104248, end_date=20231018T104248
[2023-10-18T10:42:48.273+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 93 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1576)
[2023-10-18T10:42:48.316+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:42:48.343+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:17:20.602+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-31T00:00:00+00:00 [queued]>
[2023-10-18T11:17:20.613+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-31T00:00:00+00:00 [queued]>
[2023-10-18T11:17:20.613+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:17:20.624+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-31 00:00:00+00:00
[2023-10-18T11:17:20.632+0000] {standard_task_runner.py:57} INFO - Started process 413 to run task
[2023-10-18T11:17:20.634+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-31T00:00:00+00:00', '--job-id', '141', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmppira_xfb']
[2023-10-18T11:17:20.635+0000] {standard_task_runner.py:85} INFO - Job 141: Subtask waiting_for_data
[2023-10-18T11:17:20.684+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-31T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:17:20.763+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-31T00:00:00+00:00'
[2023-10-18T11:17:20.772+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:17:20.772+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:17:20.773+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:17:20.773+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:17:20.782+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210331T000000, start_date=20231018T111720, end_date=20231018T111720
[2023-10-18T11:17:20.806+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:17:20.837+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:12:00.145+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-31T00:00:00+00:00 [queued]>
[2023-10-19T12:12:00.159+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-31T00:00:00+00:00 [queued]>
[2023-10-19T12:12:00.160+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:12:00.177+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-31 00:00:00+00:00
[2023-10-19T12:12:00.184+0000] {standard_task_runner.py:57} INFO - Started process 462 to run task
[2023-10-19T12:12:00.190+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-31T00:00:00+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpao4rbsm5']
[2023-10-19T12:12:00.192+0000] {standard_task_runner.py:85} INFO - Job 138: Subtask waiting_for_data
[2023-10-19T12:12:00.250+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-31T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:12:00.324+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-31T00:00:00+00:00'
[2023-10-19T12:12:00.333+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:12:00.333+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:12:00.334+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:12:00.334+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:12:00.346+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210331T000000, start_date=20231019T121200, end_date=20231019T121200
[2023-10-19T12:12:00.406+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:12:00.453+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
